.rubocop_todo.yml

```diff
@@ -38,7 +38,7 @@ Metrics/MethodLength:
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 304
+  Max: 305
```

Comment:
🏆

Reply:
Haha. Hopefully it wasn't too painful figuring out that you had to do this. RuboCop as a whole is _relatively_ new. We rolled through at one point and killed a bunch of the RuboCop TODOs, but all the slightly harder ones are still left.